src/status_im/utils/utils.cljs

```diff
@@ -133,3 +133,8 @@
         (map (fn [[k v]]
                [k (f v)]))
         m))
+
+(defn deep-merge [& maps]
```

Comment:
just wondering we don't have this

Reply:
@flexsurfer i didn't find it in project or in clojure.core. Proposed patch exists though https://dev.clojure.org/jira/secure/attachment/13134/CLJ-1468-deep-merge-01.patch

By the way, one-level deep merge might be done with `merge-with merge`, but not deeper.

Reply:
Can you add basic test and some doc for this utility function?

Reply:
@jeluard sure, will add.

Reply:
@jeluard added, please take a look. i'm ready to squash into one commit.